smokerd.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=GoodData Smoke testing daemon
+
+[Service]
+Type=simple
+
+Environment=CONFIG=/etc/smokerd/smokerd.yaml
+Environment='OPTIONS=-v --foreground --no-console'
```

Comment:
why --foreground? 
Couldn't we use Type=forking? Note smoker will spawn child processes when plugins are executed.